### DISCUSSION TITLE
chroot login

### DISCUSSION THREAD

**Discussion by MTollurido** [ltsp/ltsp#267] [Q&A]
Hi all! First thanks a lot to Alkis and all developers for LTSP! I use LTSP in several orphanages in Africa, students love it!
My problem: I set up the server using ubuntu mate 18.04.04 64 bit and a chroot following the tutorial in  https://github.com/ltsp/community/wiki/chroots.
All is ok, but when I try to use autologin the client boots until login window and then tries to login several times without success. 
Without autologin all is ok.
The lts.conf contanis:
[clients]
[9c:b6:54:a2:7c:bc]
HOSTNAME=pc03
AUTOLOGIN=student03
RELOGIN=1
PASSWORDS_PC03="student03/c3R1ZGVudDAzCg=="

Thanks a lot!
Mauro

**Comment by alkisg** [comment]
Hi Mauro,

you're probably missing the ltsp and sshfs packages inside the chroot. Install them and run `ltsp image` afterwards.

To get all the PAM error message shown on screen, press Ctrl+Alt+F2 in a client, and try to login as the student there.

**Comment by MTollurido** [comment]
Yes!! That was the problem. Solved.
Thanks a lot
